docs/blog/newsletter-december-2022.rst

```diff
@@ -0,0 +1,121 @@
+.. post:: December 07, 2022
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – December 2022
+#########################################
+
+Hello to all the lovely documentarians out there. The end of the year is approaching, which is a wonderful time to reflect on all the things we've done in 2022. And on what to look forward to for next year. I hope your personal list of accomplishments is long and your list of remaining things to do isn't quite as daunting as mine.
+
+This year, Write the Docs has already organized two conferences and is in the process of a third, the `Write the Docs Australia conference </conf/australia/2022/>`__. The new ideas may already be flowing there by the time you read this. And there's still time to contribute to the `2022 salary survey <https://salary-survey.writethedocs.org/>`__. The results from the survey are always helpful in understanding our industry and each of our places in it.
+
+Looking forward at next year, a `venue has been chosen and more details settled for the Portland conference <https://www.writethedocs.org/conf/portland/2023/news/announcing-cfp/>`__. The conference will once again be in person. If you have ideas you'd like to share, the `Call for Proposals <https://www.writethedocs.org/conf/portland/2023/cfp/>`__ is open until the beginning of February. The call is open to documentarians of all stripes, so don't be afraid to send in your proposal.
+
```

Comment:
Let's add a line here that the Atlantic CFP will up sometime Q1 2023 ish? To forestall more questions